index.js

```diff
@@ -107,6 +111,8 @@ instance.prototype.init = function () {
 
 	self.addSystemCallback('graphics_indicate_push', self.bank_indicate_push.bind(self))
 
+	self.addSystemCallback('bank_pressed', self.bank_pressed.bind(self))
```

Comment:
Understood. I'm not sure if there's a better one, but I split this between db_set and then added a feedback check to the existing devices_list function. In my testing, this covers: new devices, reconnecting devices, and setting pages via settings.

Reply:
Honestly, I like `db_set` even less.. 
In part because when it comes to 2.3, this is going to have to be fixed to be done properly. Im not sure if its worth trying to implement an event for this to be done from the right places now, or to leave this inefficient implementation in place for 2.2